### DISCUSSION TITLE
Plotter and a mats question

### DISCUSSION THREAD

**Discussion by jeagh** [pybricks/support#974] [Q&A]
I'm attempting to build a plotter with the pieces available in the Robot Inventor kit. I have the software sorted, using a gcode library but I'm having difficulty with the mechanical side. To make matters worse I'm currently in an area with very poor mobile phone coverage and the weather is overcast so I have limited power to play with the computer.

I've experimented with the following idea and the amount of play in the set up makes, I think, the whole project unworkable. I have two beams in an "A" arrangement as follows and I'd like to compensate for the non linearity of movement.

   /\
  /  \
 /    \

The lower end of the left hand beam rotates through a pair of gears, the upper end of the beam has a pivot point that connects to the upper end of the right hand beam. A pen is attached to the lower end of that beam.

I don't have a protractor and I cannot spare the power to experiment too much until maybe next week.

I've been trying to calculate how far the pen will move per degree of motor rotation and a method to calculate how many degrees of rotation will be required to move the pen the desired amount given the non linearity in the design.

Any suggestions will be greatly appreciated.

Now to see if I can send this message.

I see my diagram hasn't displayed properly. The two beams are in an "A" shape. I hope the helps.

**Comment by laurensvalk** [comment]
The pictures didn't seem to have come through. Could this provide some inspiration? https://www.youtube.com/watch?v=fGQu90EPVAM

**Reply by jeagh** [reply]
Thank you Laurens, I cannot get the YouTube page to load because of my poor data rate at the moment, however, I know which plotter the link refers to. I'll have to see if I can find a supplier for the tracks and long shafts plus some of the extra parts required.
